src/Evervault/EvervaultHttp.php

```diff
@@ -155,7 +169,7 @@ private function _makefunctionRunRequest($functionName, $body = [], $headers = [
         return $this->_handleApiResponse($this->curl, $response, $headers);
     }
 
-    public function runfunction($functionName, $functionData, $additionalHeaders) {
+    public function runFunction($functionName, $functionData, $additionalHeaders) {
```

Comment:
```suggestion
    public function runFunction($functionName, $functionData, $additionalHeaders) {
```

How did this work before fixing the capitalisation? (or is PHP that loose...)